pkgs/applications/window-managers/jwm/default.nix

```diff
@@ -38,12 +36,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  preConfigure = "./autogen.sh";
-
   meta = {
     homepage = "http://joewing.net/projects/jwm/";
     description = "Joe's Window Manager is a light-weight X11 window manager";
-    license = lib.licenses.gpl2;
+    license = lib.licenses.gpl2Plus;
```

Comment:
Seems to be using MIT license instead :thinking: .

https://github.com/joewing/jwm/blob/v2.4.0/LICENSE

Reply:
Issue of license change: https://github.com/joewing/jwm/issues/320

Reply:
Thanks for pointing it out.
Fixed.